README.md

```diff
@@ -211,6 +211,7 @@ For the moment, we are currently focused on the CISA KEV Database and Google Tsu
 | CVE-2023-20269                                          | ❌           | Not remotely exploitable/User interaction needed.      |
 | CVE-2023-20198                                          | ✅           | Nuclei Template was used.                              |
 | CVE-2023-20109                                          | ❌           | Not remotely exploitable/User interaction needed.      |
+| CVE-2023-2780                                           | ✅           | Official Nuclei template.                              |
```

Comment:
Why not keep it as the others : "Nuclei Template was used."

Reply:
just saw the update

Reply:
> Why not keep it as the others : "Nuclei Template was used."

We need to differentiate between official template and custom templates we wrote.